apb/templates/broker-user-auth.yaml

```diff
@@ -0,0 +1,17 @@
+---
+kind: ClusterRole
```

Comment:
The only thing I don't see is this template being used in deployment.

Reply:
I am very confused by this statement sorry can you explain what I need to do?

Reply:
I would expect to see `broker-user-auth.yaml` somewhere in this list (https://github.com/shawn-hurley/ansible-service-broker/blob/a85d2a365ac1fb293dae0951ee61320038649523/apb/tasks/main.yml#L54) in order for the `broker-user-auth ClusterRole` to be created.